cpp/src/rolling/rolling.cu

```diff
@@ -50,6 +50,10 @@
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/traits.hpp>
 #include <memory>
+#include "rmm/thrust_rmm_allocator.h"
+#include "thrust/execution_policy.h"
+#include "thrust/find.h"
+#include "thrust/transform.h"
```

Comment:
Can you make these into `< >` style includes?

Reply:
Yessir. I've to also figure out how to tell VSCode to prefer angle-brackets, even for user headers.